lib/minitest/reporters/base_reporter.rb

```diff
@@ -86,8 +86,12 @@ def print(*args)
       end
 
       def print_info(e, name=true)
-        print "#{e.exception.class.to_s}: " if name
-        e.message.each_line { |line| print_with_info_padding(line) }
+        if name
+          print "#{e.exception.class.to_s}: "
+          e.exception.message.each_line { |line| print_with_info_padding(line) }
+        else
+          e.message.each_line { |line| print_with_info_padding(line) }
```

Comment:
Had thought about it.. little shitty checking for `e`'s class like that, but I'm okay coz `print_info` may as well have been used as you mentioned. Code updated. Lets merge it soon! Seeing exception backtrance printed twice each time is driving me crazy (esp. for long backtraces)..